src/utils.ts

```diff
@@ -1,7 +1,21 @@
-import { PubSubConfig } from './types';
+import { PubSubConfig, PubSubRequestType } from './types';
 
 export const makePubSubConfig = <Data, Context>(
   data: PubSubConfig<Data, Context>,
 ): PubSubConfig<Data, Context> => {
   return data;
 };
+
+export const makeMockPubSubMessage = (
+  payload: unknown,
+  jsonStringify = true,
+): PubSubRequestType => {
+  const data = Buffer.from(
+    jsonStringify ? JSON.stringify(payload) : (payload as string),
```

Comment:
I would maybe do a `typeof` check here. Make sure the payload is string.
Or have the payload be an object of `{stringData: string} | {jsonData: unknown}`

Not super important, just try to make as much as possible to crash at compile time